README.md

```diff
@@ -15,29 +15,29 @@ with Menge.  Change to the directory: `$MENGE_ROOT$\Exe`.  Once there, you have
 ..* Windows: `menge -p ..\examples\core\4square.xml`
 ... This will execute the 4square example with all of the built-in default values.
 
-You can see the available command-line parameters by typing:
+You can see the command-line parameters by typing:
 
-`menge -h` in Windos or `./menge -h` in Linux.
+`menge -h` in Windows or `./menge -h` in Linux.
 
-While a simulation is running, a visualization window will be displayed.
+While a simulation is running, a visualization window will display.
```

Comment:
"display" is properly a transitive verb.  https://www.merriam-webster.com/dictionary/display